projects/angular-cld/src/lib/cloudinary-image-encode-sdk.spec.ts

```diff
@@ -0,0 +1,215 @@
+import { Component, DebugElement } from '@angular/core';
```

Comment:
What the file name means? why not cloudinary-sdk-analytics or something similiar?

Reply:
Will change to cloudinary-sdk-analytics-spec

@asisayag2 See the comment for the PR. Test are failing bc we use cloudinary-core to generate the analytics. We'll need to release core with the changes and updated the core version on the angular sdk. That being said, when tested locally with the change in core the tests pass